### DISCUSSION TITLE
Choose Max or min for measurable habits

### DISCUSSION THREAD

**Discussion by harishbewe** [iSoron/uhabits#1398] [Feature Requests] [answered]
In measurable values, For negative habits, an option to set max value and minimum value to positive habits will be a good option. Like if I am planning to reduce my screen time, now there is only option to note the amount of hours. If I can set a max value and the streak might seem purposeful

**Comment by hiqua** [comment, marked as answer]
That's already implemented on `dev`, coming in the next release https://github.com/iSoron/uhabits/pull/1372/commits/b66a6ff71704fd0dbc1a4812a7104c98e168c9a3

**Reply by harishbewe** [reply]
Oh! That's nice to hear. Thank you.

**Reply by Henwij** [reply]
Glad to hear that!!!

**Reply by b5e6ue6n8ew4q7wbqvc2** [reply]
When?

**Reply by hiqua** [reply]
Available as a beta on the play store, available on F-Droid.
